pinot/regressors/biophysical_regressor.py

```diff
@@ -23,95 +24,103 @@ class BiophysicalRegressor(torch.nn.Module):
 
     """
 
-    def __init__(self, base_regressor=None, *args, **kwargs):
+    def __init__(self, base_regressor_class=None, *args, **kwargs):
         super(BiophysicalRegressor, self).__init__()
-        self.base_regressor = base_regressor
+        # get the base regressor
+        self.base_regressor_class = base_regressor_class
+        self.base_regressor = base_regressor_class(
+                *args, **kwargs
+        )
+
+        # initialize measurement parameter
         self.log_sigma_measurement = torch.nn.Parameter(torch.zeros(1))
 
-    def g(self, func_value=None, test_ligand_concentration=1e-3):
-        return 1 / (1 + torch.exp(-func_value) / test_ligand_concentration)
+    def _get_measurement(self, delta_g, concentration=1e-3):
```

Comment:
@yuanqing-wang : What units are you using here? It would be good to document them in the docstring!

Reply:
that's my code, I will add them